week08_pomdp/practice_tensorflow.ipynb

```diff
@@ -245,8 +260,9 @@
     "\n",
     "        total_reward = 0\n",
     "        while True:\n",
-    "            new_memories, readouts = agent.step(\n",
-    "                prev_memories, observation[None, ...])\n",
+    "            observations = observation[np.newaxis, :].astype('float32')\n",
```

Comment:
I don't think there should be both `observations` and `observation`. The easiest solution is to inline `observations`.

Reply:
ok